### PR TITLE
Add CI job to run test suite

### PR DESCRIPTION
**Pull request by alerque**
See comments in #32

### CODE REVIEW

File: run_tests.sh
Comment (telemachus) [inline run_tests.sh:11]: I don't like this part of the original test setup. It looks for
dependencies in the user's normal vim environment.

That said, as far as I can tell, this file is not used by the CI.
Is that right? if so, I can easily edit this file later for local
test runs.

Does that make sense to you?

**Review by telemachus — APPROVED** [review]
This all looks great to me.

My only questions are about the file `run_tests.sh`.
But since I don't think that the CI uses it, we can
probably edit that further later.

File: run_tests.sh
Comment (alerque) [reply run_tests.sh:11]: Yes I don't like this either, and no it is not used by CI. I only messed with it to fix paths that I was changing not to actually overhaul it. It isn't even complete because it doesn't account for another dependency needed for tests (vim-surround). I would suggest something completely different as a local test runner anyway.

**Comment by telemachus** [comment]
I don't think I understand the "pending review" workflow at all.

Sorry to be dense, but do you want me to review this further?
I can see that it needs one review to be merged, but I also see
that you dismissed my earlier review. Let me know if you want
another set of eyes on anything.

**Comment by alerque** [comment]
Sorry, the GitHub PR workflow works fine for some situations and less so for others. I've been trying to set it up to ease outside contributors getting their stuff looked at, but it's a bit noisy and even in the way for two active project contributors.

The fact that a review is requested is automatic just by opening a PR. Also the fact that I pushed an extra commit here dismissed your previous review and re-requested it automatically. I don't really think you need to dump effort into reviewing my addition of a CI badge to the readme ;-)

If it gets in the way dealing with outside contributions let me know and I'll fiddle with settings.

**Comment by telemachus** [comment]
> I don't really think you need to dump effort into reviewing my addition of a CI badge to the readme ;-)

It was a great badge: I definitely approve. More seriously, thanks for clarifying re review requests.

**Pull request merged**
